firefox/janitor-git.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Firefox (git)",
+  "description": "The Firefox browser (git flavour)",
+  "icon": "https://janitor.technology/img/firefox.svg",
+  "docker": {
+    "image": "JanitorTechnology/firefox-git:base"
+  },
+  "ports": {
+    "22": {
+      "proxy": "none"
+    },
+    "8088": {
+      "proxy": "https",
+      "label": "VNC"
+    },
+    "8089": {
+      "proxy": "https",
+      "label": "Cloud9"
+    }
+  },
+  "previewUrls": {
+    "VNC": "https://${HOSTNAME}/${CONTAINER}/8089/vnc.html"
```

Comment:
Nit: didn't we agree to remove "previewUrls" altogether, because they duplicate what's in "ports"?